### DISCUSSION TITLE
Vercel support cases deleted -- account security

### DISCUSSION THREAD

**Discussion by avaer** [vercel/community#4097] [Help]
### Summary

I've been trying to fix a deployment issue where my private repository code is being leaked to a known hacked vercel account/email. I want to remove my repositories from that hacked account, as a matter of private code security of my repository here on GitHub.

I tried to remove/re-add vercel integration on the GitHub side and the issue persists. This seems to be a security hole on the Vercel side.

I've filed 3 support cases (on my paid plan) and all 3 times the support cases simply vanished into thin air (my account shows no support cases).

Here is one such support case: https://vercel.com/teams/webmixedreality-s-team/support/cases/00158470

### Example

https://vercel.com/teams/webmixedreality-s-team/support/cases/00158470

### Steps to Reproduce

- Open a support case
- It will disappear after an hour or two

**Comment by avaer** [comment]
Is there some way to contact support about my code being deployed to a hacked vercel account due to a Vercel misconfguration?

**Comment by amyegan** [comment]
Hi @avaer. Thanks for reporting this. I just checked on that case number and I still see with an "Open" status in the queue. 

I'm reaching out to the Support Center team for help investigating why you can't see your open cases.

**Reply by avaer** [reply]
What is the response from the Support Center team re: this code security issue?

**Comment by avaer** [comment]
This is what I see, so I can only assume the cases are being wiped from existence...

<img width="1279" alt="Screenshot 2023-09-16 at 5 12 27 AM" src="https://github.com/vercel/community/assets/6926057/86677161-faf1-4519-83ea-2c383dead05b">

**Comment by avaer** [comment]
@amyegan Anything new here? I still don't see my support cases and this is a serious security issue.

**Comment by avaer** [comment]
I found this for anyone experiencing similar issues with code security and support on Vercel: https://developers.cloudflare.com/pages/migrations/migrating-from-vercel/

Apparently Cloudflare has a decent migration path.
